app/views/govuk_publishing_components/components/docs/organisation_logo.yml

```diff
@@ -182,3 +182,13 @@ examples:
         brand: cabinet-office
         crest: 'single-identity'
       inline: true
+  on_a_dark_background:
+    data:
+      organisation:
+        name: "Prime Minister's Office,<br>10 Downing Street"
+        url: '/government/organisations/prime-ministers-office-10-downing-street'
+        brand: 'prime-ministers-office-10-downing-street'
+        crest: 'no10'
```

Comment:
I assume you want `eo` instead of `no10` here?

Reply:
I think we might want to retain `no10` here - have messaged you on Slack.